requirements.txt

```diff
@@ -1,5 +1,5 @@
 grpcio==1.30.0
-protobuf==3.15.0
+protobuf==3.18.3
```

Comment:
Should we just tear out protobufs at this point? We don't use this anywhere anymore. Perhaps delay this for deboggling month?

Reply:
I think just removing this is fine.